tutorial.ipynb

```diff
@@ -489,6 +485,237 @@
+   "source": [
+    "## Simulating - advanced"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are a number of advanced features that can be used in simulation. A few of these will be described below. Detail can also be found in the documentation here: https://nasa.github.io/progpy/prog_models_guide.html#simulation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Saving results at specific points"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Earlier, we demonstrated how save_freq specifies the frequency at which the results are saved in simulation. There are occasionally circumstances when you need the results at a specific time (e.g., check points, waypoints, etc.). This is accomplished with the save_pts argument in simulation.\n",
+    "\n",
+    "For example, in the following simple case, we're reusing the example from earlier, but we're saving the results at 1 second, 5 seconds, 6 seconds, and 7 seconds. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Reset noise\n",
+    "m.parameters['process_noise'] = False\n",
+    "m.parameters['measurement_noise'] = False\n",
+    "\n",
+    "# Simulate to a threshold\n",
+    "(times, _, states, outputs, _) = m.simulate_to_threshold(future_load, dt=config['dt'], threshold_keys='impact', save_pts=[1, 5, 6, 7])\n",
+    "\n",
+    "# Print Results\n",
+    "print('states:')\n",
+    "for (t,x) in zip(times, states):\n",
+    "    print('\\t{:.2f}s: {}'.format(t, x))\n",
+    "\n",
+    "# Plot results\n",
+    "states.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that now we only have the data at the points specified (plus initial and final). As a result, the graph looks lopsided.\n",
+    "\n",
+    "save_pts can also be used to adjust the save frequency as degradation progresses, for example by having more concentrated save pts around the end of degradation, like the example below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Simulate to a threshold\n",
+    "(times, _, states, outputs, _) = m.simulate_to_threshold(future_load, dt=config['dt'], threshold_keys='impact', save_pts=[1, 2, 3, 4, 5, 5.5, 6, 6.25, 6.5, 6.75, 7, 7.1, 7.2, 7.3, 7.4, 7.5, 7.6, 7.7, 7.8])\n",
+    "\n",
+    "# Print Results\n",
+    "print('states:')\n",
+    "for (t,x) in zip(times, states):\n",
+    "    print('\\t{:.2f}s: {}'.format(t, x))\n",
+    "\n",
+    "# Plot results\n",
+    "states.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now the curve is much better defined as the simulation progresses. This is frequently of interest since users are mostly interested in behavior close to the event occurance."
```

Comment:
```suggestion
    "Now the curve is much better defined as the simulation progresses. This is frequently of interest since users are mostly interested in behavior close to the event occurence."
```